src/main/resources/GoogleApps/Groovy.xml

```diff
@@ -410,7 +410,7 @@ public class GoogleAppsGroovy {
             try {
                 // Force main wiki database to create the user as global
                 context.setDatabase("xwiki")
-                email = (user.emails!=null &amp;&amp; user.emails.size()&gt;0) ? user.emails.getAt(0).value : "";
+                email = (user.emails!=null &amp;&amp; user.emails.size()>0) ? user.emails.getAt(0).value : "";
```

Comment:
Is this change the result of xar page export? As `&` was resolved to `&amp;`, so the `>` should be resolved as `&gt;`.

Reply:
fixed.